.github/workflows/workflow.yml

```diff
@@ -13,41 +13,39 @@ jobs:
           - macos-latest
           - ubuntu-latest
         ocaml-version:
-          - 4.10.0
+          - 4.10.1
           - 4.09.1
           - 4.08.1
-          - 4.08.0
-          - 4.07.0
+          - 4.08.1
```

Comment:
This adds 4.08.1 a second time. I think this is an error.

Reply:
Indeed... I will fix it.